plugins/commando.c

```diff
@@ -42,11 +42,16 @@ struct commando {
 	const char *json_id;
 };
 
+struct blacklist {
```

Comment:
You should try to avoid a non-working commit.  I would suggest combining this one and the next one?

Also, commit title should be `commando: add blacklist command`